packages/e2e-test-utils/src/has-php-error.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Regular expression matching a displayed PHP error within a markup string.
+ *
+ * @link https://github.com/php/php-src/blob/598175e/main/main.c#L1257-L1297
+ *
+ * @type RegExp
+ */
+const REGEXP_PHP_ERROR = /<b>(Fatal error|Recoverable fatal error|Warning|Parse error|Notice|Strict Standards|Deprecated|Unknown error)<\/b>/;
```

Comment:
Should this match be more detailed? `<b>Notice</b>` or `<b>Warning</b>` might be too common and cause false positives.  We could at least add `:` after closing tag:
https://github.com/php/php-src/blob/598175e/main/main.c#L1321

Reply:
Yeah, I suppose it risks false positives in its current form. With the placeholder formatting and multiple variants, it's harder to get a good pattern to cover all the error outputs. I agree the colon `:` and even the subsequent double-space `  ` could be a good improvement to eliminate those false positives. I'll make the change.

Reply:
Thank you, I will give it a spin as code wise it look good 👍